src/components/ManageUsers/manage-users.css

```
@@ -0,0 +1,84 @@
+.usersearch__h1 {
+  font-size: 16px;
+  color: #e3893e;
+  display: right;
+  font-weight: 200px;
```

`font-weight` has no pixels. On figma the text is in semi-bold so the corresponding value is 600
```suggestion
  font-weight: 600;
```

---

Already changed that in latest commit